jetstream/metadata.py

```diff
@@ -113,7 +113,10 @@ def from_config(
                 != config.experiment.experiment.proposed_enrollment
                 else None,
                 skip=config.experiment.skip,
-                url=DEFAULT_CONFIG_REPO + "/blob/main/" + config.experiment.normandy_slug + ".toml",
+                url=METRIC_HUB_REPO
+                + "/blob/main/jetstream/"
+                + config.experiment.normandy_slug
+                + ".toml",
```

Comment:
@mikewilli  I realize that when we move these configs, this URL will be invalid for old experiments. Is this used for anything critical on the Experimenter side?

Reply:
There is some text we put on the Results page if the experiment has external configs, which uses this URL field to link to the experiment's config file. Probably not critical.

Wouldn't this only update experiments that have external configs to the correct external config URL though? Or do you mean that the older experiments will have an invalid URL when `jetstream-config` is removed (in which case, I think we could just archive that repo so the links remain valid)? Might be misunderstanding what you're saying.

Reply:
I was thinking of the latter case (jetstream-config being gone), but you are right, we can archive the repo